WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -28130,7 +28130,7 @@
 	culture = old_slavic
 }
 1042167 = {
-	name="Atillid" 
+	name="the Hun" 
 	culture = hunnic 
```

Comment:
We can script in nicknames but the dynasty name stays because it is more appropriate and that it is used for the whole dynasty, not just Attila himself. Also "the Hun" is in English and not good from the optic of supporting more than one language (this is the kind of thing we can't translate in localisation files).